decidim-admin/app/views/decidim/admin/organization_appearance/form/_colors.html.erb

```diff
@@ -65,10 +65,11 @@
         <%= form.color_field :alert_color, value: current_organization.colors["alert"] %>
       </div>
 
```

Comment:
The div should be closed here. 

I don't make the suggestion because we'd need to also indent the next lines

Reply:
Actually I think it is closed correctly. 

If we refer to develop branch, https://github.com/decidim/decidim/blob/develop/decidim-admin/app/views/decidim/admin/organization_appearance/form/_colors.html.erb  we can see that within the `details` tag opened at line 12 we have a single `div` (.color-picker__box) at line 17 that is not closed befor the `details` is being closed at line 56.

Reply:
Sorry to not be clear enough: the `div` you're talking about is perfect! The other `div` (line 73) should be closed here (line 67). The UI is minimally broken if not (as we have a `div.row.colum` inside another `div.row.column`)

Reply:
Ah, so you say that the div that is closed at line 73, should in fact be closed above the if ( ~ 67) . 
Chekd if 7f4c0bb304 is the thing you wanted.